qemu/tests/block_copy.py

```diff
@@ -499,3 +499,46 @@ def verify_md5s(self):
         file_names = self.params["file_names"].split()
         for name in file_names:
             self.verify_md5(name)
+
+    def mount_disks(self):
+        """
+        Mount added disks inside guest(only for linux).
+        """
+        os_type = self.params["os_type"]
+        if os_type == "linux":
+            session = self.get_session()
+            fs_type = self.params.get("mount_fstype", "ext4")
+            disks = utils_misc.list_linux_guest_disks(session)
+            for disk in disks:
+                if not re.search("da", disk):
+                    mount_point = "/mnt/%s" % disk.split('/')[2]
+                    utils_misc.mount(disk, mount_point, fs_type, session=session)
```

Comment:
Line 515, 519 and 533 are too long.

You may want to use:
```python
if re.search("da", disk):
    continue
mount_point = "/mnt/%s" % disk.split('/')[2]
```
to save some indentations.

Reply:
The rest is ok to me